example/serialized_request/README.md

```diff
@@ -0,0 +1,44 @@
+# A Guide on How to Execute a Serialized Request in Client SDKs
+This guide briefly explains how to execute a serialized request in client SDKs by looking at how Node SDK can execute a request created by Web SDK.
+
+## Prerequisite
+The diagram below shows the whole architecture of this guide.
+
+![](img/samplecode.png)
+
+Web client is the client side (for example browser or mobile devices) of a product/service that is
```

Comment:
Sorry to say this but I don't think it's well written since probably mixing up what they are, what they do in this guide and some requirements that this sample code can work.